src/solver/conv_hip_implicit_gemm_bwd_data_xdlops.cpp

```diff
@@ -335,7 +336,7 @@ bool ConvHipImplicitGemmBwdXdlops::IsApplicable(const ConvolutionContext& ctx,
     if(!problem.IsLayoutNHWC())
         return false;
     const std::string& arch = ctx.GetStream().GetDeviceName();
-    if( arch != "gfx908" || arch != "gfx90a" )
+    if(arch != "gfx908" && arch != "gfx90a")
```

Comment:
[Recommendation] I think this improves readability, but the final decision is up to you.
```suggestion
    if(!(arch == "gfx908" || arch == "gfx90a"))
```